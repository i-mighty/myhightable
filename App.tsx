/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

import React from 'react';
import IntroContainer from '@src/containers/AppIntro/Container';
import { StyleProvider } from 'native-base';
import getTheme from '@src/native-base-theme/components';
import platform from '@src/native-base-theme/variables/platform';
import { ThemeProvider } from 'styled-components/native';
import theme from '@src/stores/theme';

const App: React.FC = () => (
  <StyleProvider style={getTheme(platform)}>
    <ThemeProvider theme={theme}>
      <IntroContainer />
    </ThemeProvider>
  </StyleProvider>
);

export default App;
