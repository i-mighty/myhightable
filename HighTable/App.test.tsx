import React from 'react';
import { render } from '@testing-library/react-native';
import App from './App';

test('It renders successfully', async () => {
  const { getByTestId, baseElement } = render(<App />);

  expect(getByTestId('title').children.join('')).toBe('HighTable');
  expect(baseElement).toMatchSnapshot();
});
