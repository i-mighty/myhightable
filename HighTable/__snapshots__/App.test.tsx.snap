// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples of some things 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <SafeAreaView>
      <ScrollView
        contentInsetAdjustmentBehavior="automatic"
      >
        <View>
          <Text
            testID="title"
          >
            HighTable
          </Text>
        </View>
      </ScrollView>
    </SafeAreaView>
  </View>
</View>
`;
